Docs: Add Installation YouTube Videos to documentation pages. (#88443)
grafana/grafana: docs/sources/setup-grafana/installation/windows/index.md

# Install Grafana on Windows

> The following video guides you through the steps in which you can install Grafana in Windows standalone installer as described in this document:

Reviewer: ```suggestion
The following video guides you through the steps you use to install Grafana in Windows standalone installer as described in this document:
```
